code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -173,4 +173,4 @@
 	name = "Gloves of Headpats"
 	desc = "You feel the irresistable urge to give headpats by merely glimpsing these."
 	accepted_intents = list(INTENT_HELP)
-	click_speed_modifier = 0 // That's some serious headpatting
\ No newline at end of file
+	click_speed_modifier = 6 // That's some serious headpatting
```

Comment:
just get rid of this line and let inheritance handle what the modifier is for the original gloves of the north star

Reply:
thats still too fast

Reply:
There's virtually no point to it being 6.

You can make a *gun* click faster than this.

Reply:
Agreed with above

Reply:
Might I again, also suggest, this be an arcade reward? Not horrifyingly expensive, but enough that it's not something that shows up 5 minutes in, every shift, and peopel can make hundreds of them.